bin/pycbc_make_sky_grid

```diff
@@ -0,0 +1,112 @@
+
+This sky grid will be used by `pycbc_multi_inspiral` to find multi-detector gravitational wave triggers and calculate the
+coherent SNRs and related statistics. 
+
+The grid is constructed following the method described in Section V of https://arxiv.org/abs/1410.6042"""
+
+import numpy as np
+import h5py
+import argparse
+from pycbc.detector import Detector
+import itertools
+from scipy.spatial.transform import Rotation as R
+
+def spher_to_cart(sky_points):
+    """Convert spherical coordinates to cartesian coordinates.
+    """
+    cart = np.zeros((len(sky_points), 3))
+    cart[:,0] = np.cos(sky_points[:,0]) * np.cos(sky_points[:,1])
+    cart[:,1] = np.sin(sky_points[:,0]) * np.cos(sky_points[:,1])
+    cart[:,2] = np.sin(sky_points[:,1])
+    return cart
+
+def cart_to_spher(sky_points):
+    """Convert cartesian coordinates to spherical coordinates.
+    """
+    spher = np.zeros((len(sky_points), 2))
+    spher[:,0] = np.arctan2(sky_points[:,1], sky_points[:,0])
+    spher[:,1] = np.arcsin(sky_points[:,2])
+    return spher
+
+parser = argparse.ArgumentParser(description=__doc__)
+parser.add_argument('--ra', type=float, help="Right ascension (in rad) of the center of the external trigger error box")
+parser.add_argument('--dec', type=float, help="Declination (in rad) of the center of the external trigger error box")
+parser.add_argument('--instruments', nargs="+", type=str, required=True, help="List of instruments to analyze.")
+parser.add_argument('--sky-error', type=float, required=True, help="3-sigma confidence radius (in rad) of the external trigger error box")
+parser.add_argument('--trigger-time', type=int, required=True, help="Time (in s) of the external trigger")
+parser.add_argument('--timing-uncertainty', type=float, default=0.0001, help="Timing uncertainty (in s) we are willing to accept")
+parser.add_argument('--output', type=str, required=True, help="Name of the sky grid")
+
+args = parser.parse_args()
+
+if len(args.instruments) == 1:
+    parser.error('Can not make a sky grid for only one detector.')
+
+args.instruments.sort() # Put the ifos in alphabetical order
+detectors = args.instruments
+detectors = [Detector(d) for d in detectors]
+detector_pairs = list(itertools.combinations(detectors, 2))
+
+# Select the detector pair for which the target location has smallest relative arrival time difference
+tds = [detector_pairs[i][0].time_delay_from_detector(detector_pairs[i][1], args.ra, args.dec, args.trigger_time) for i in range(len(detector_pairs))]
+smallest_td = np.argmin(tds)
+
+# Calculate the light travel time between the two detectors
+t = detector_pairs[smallest_td][0].light_travel_time_to_detector(detector_pairs[smallest_td][1])
+
+# Calculate the required angular spacing of the sky points
+angular_spacing = (2*args.timing_uncertainty) / np.sqrt(t**2 - tds[smallest_td]**2)
+
+sky_points = np.zeros((1, 2))
+
+number_of_rings = int(args.sky_error / angular_spacing)
```

Comment:
This should use `math.ceil()` instead of `int()`. If the sky error is very very small (e.g. a Swift/BAT GRB with a localization much more precise than what is possible with LVK) then you still want at least one sky point in the grid.

Reply:
Ah but you use `number_of_rings + 1` below, so `int` is fine.